src/square.js

```diff
@@ -1 +1,5 @@
 'use strict';
+
+const square = (x) => x ** 2;
+
+module.exports.square = square;
```

Comment:
```suggestion
module.exports.square = (x) => x ** 2;
```
You can do it even simpler and shorter.